### PR TITLE
remove Handcraft mixer recipes.

### PR DESCRIPTION
**Pull request by Dream-Master**
* https://github.com/GTNewHorizons/NewHorizonsCoreMod/pull/218
* https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/7606

### CODE REVIEW

**Comment by Elisis** [comment]
Could you make the HSS-G mixer recipe EV too?

**Comment by Dream-Master** [comment]
Will do

File: src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java [hunk 1]
```diff
@@ -407,8 +407,8 @@ public GT_MetaGenerated_Item_01() {
...
         ItemList.Battery_Hull_HV.set(addItem(502, "Large Battery Hull", "An empty HV Battery Hull", new ItemData(Materials.BatteryAlloy, OrePrefixes.plate.mMaterialAmount * 9L), new TC_Aspects.TC_AspectStack(TC_Aspects.ELECTRUM, 4L), new TC_Aspects.TC_AspectStack(TC_Aspects.METALLUM, 4L), new TC_Aspects.TC_AspectStack(TC_Aspects.VACUOS, 1L)));
 
-        GT_ModHandler.addCraftingRecipe(ItemList.Battery_Hull_LV.get(1L), GT_ModHandler.RecipeBits.NOT_REMOVABLE, new Object[]{"C", "P", "P", 'P', OrePrefixes.plate.get(Materials.BatteryAlloy), 'C', OreDictNames.craftingWireTin});
-        GT_ModHandler.addCraftingRecipe(ItemList.Battery_Hull_MV.get(1L), GT_ModHandler.RecipeBits.NOT_REMOVABLE, new Object[]{"C C", "PPP", "PPP", 'P', OrePrefixes.plate.get(Materials.BatteryAlloy), 'C', OreDictNames.craftingWireCopper});
+        GT_ModHandler.addCraftingRecipe(ItemList.Battery_Hull_LV.get(1L), GT_ModHandler.RecipeBits.NOT_REMOVABLE, new Object[]{"Cf ", "Ph ", "Ps ", 'P', OrePrefixes.plate.get(Materials.BatteryAlloy), 'C', OreDictNames.craftingWireTin});
```
Comment (Elisis) [inline src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java:410]: Is f, h, s not being defined intentional?

Comment (Dream-Master) [reply src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java:410]: f = file h= hammer etc. Need to test now

Comment (Dream-Master) [reply src/main/java/gregtech/common/items/GT_MetaGenerated_Item_01.java:410]: works
image

**Review by Elisis — APPROVED** [review]

**Comment by GTNH-Afx237v7** [comment]
Somehow HSS-S dust does not show up in NEI; HSS-G and HSS-E work fine

**Pull request merged**